src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,202 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+@ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all lowercase
+    Given this school exists:
+            | schoolId     | nameOfInstitution                    | gradeLevels                   | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]           | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=ut+austin+college+of+education+graduate" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+  @ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all uppercase
+    Given a school exists:
+            | schoolId     | nameOfInstitution                                    | gradeLevels               | educationOrganizationCategories      |
+            | 6            | UT Austin College of Education Under Graduate        | [ "Postsecondary" ]       | [ "Educator Preparation Provider" ]  |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=UT+AUSTIN+COLLEGE+OF+EDUCATION+UNDER+GRADUATE" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+
+  @ignore
+  Scenario: Ensure clients can GET information filtering by totalCount
```

Comment:
You already have this in another file, so I think you remove this scenario.

Reply:
Removed